test/system/090-events.bats

```diff
@@ -129,3 +129,69 @@ EOF
     run cat $events_file
     is "$output" ".*\"Name\":\"$IMAGE" "test"
 }
+
+function _populate_events_file() {
+    # Create 100 duplicate entries to populate the events log file.
+    # The total file size is 9500 bytes.
```

Comment:
It's actually not, and a future maintainer may be misled by thinking that there's some important reason why the file should be 9500 bytes. There isn't (AFAICT), so it's best just to remove this comment entirely. In fact, here's an alternative suggestion:
```bash
    # Create 100 duplicate entries to populate the events log file.
    local events_file=$1
    truncate --size=0 $events_file
    for i in {0..99}; do
        printf '{"Name":"busybox","Status":"pull","Time":"2022-04-06T11:26:42.7236679%02d+02:00","Type":"image","Attributes":null}\n' $i >> $events_file
    done
```
Reasons:
* There's no reason to make it a oneliner
* `truncate`, because the name `populate` suggests "make this the contents of the file" (vs "add to existing file", which is what the function does as it is right now). This then lets you remove the `rm $eventsFile` below.
* `printf` and `%02d` because it's just a good habit to create distinct log entries. (And, guess what? Doing this exposes a real bug in your test! I will let you follow up on that).